compatibility_spark-3/src/test/resources/delta-cdc-to-snapshot/test-data/second-input.csv

```diff
@@ -0,0 +1,32 @@
+id1,value27,2022-02-05 14:11:30.106,UP
```

Comment:
How is the second input different from the first one? Why can't they both be in the same file?

Reply:
I have it there to test conflicts, between delta table and new input data.